File: data/src/main/java/com/roman/kubik/spivanyklicejista/data/category/CategoryEntity.java
package com.roman.kubik.spivanyklicejista.data.category;

/**
 * Created by kubik on 1/14/18.
 */

import android.arch.persistence.room.Entity;
import android.arch.persistence.room.PrimaryKey;

@Entity(tableName = "category")
public class CategoryEntity {

    @PrimaryKey
    private int id;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }
}

File: app/src/main/java/com/roman/kubik/songer/data/local/category/CategoryEntity.kt
package com.roman.kubik.songer.data.local.category

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey


@Entity(tableName = "category")
data class CategoryEntity(
        @PrimaryKey var id: Int,
        @ColumnInfo(name = "name") var name: String)